src/manage/project-advanced.md

```diff
@@ -14,76 +14,67 @@ You can follow our [Working collaboratively](../tutorials/working-collaborativel
 :::
 
 ### Add users to a workspace
-If you want to share all projects in your workspace with a group of users, you can invite them to be a **member** of a workspace. It is also possible to invite people as **guests** and share only selected projects with them.
+If you want to share all projects in your workspace with a group of users, you can invite them to become members or guests of a workspace.
 
 You have to be the **admin** or **owner** of the workspace to manage access to the workspace. See [Member Roles and Permissions](./permissions/#workspace-member-roles) for more details.
 
 To add users to a workspace:
 1. Log into <AppDomainNameLink />
-2. Select **Members** from the left panel and click on **Add People**
-   ![Add people in Members tab in Mergin Maps](../tutorials/working-collaboratively/web-workspace-add.jpg "Add people in Members tab in Mergin Maps")
-3. In the invite form, enter the email addresses of people you want to invite. 
-   Select if you invite them as **members** or as **guests**:
-   - for **members**, select their [member role](./permissions/#workspace-member-roles). Members have access to all projects in the workspace.
-   - for **guests**, specify the projects to which they should have access. When inviting guests, you can give them [*read* or *write* permissions](./permissions/#project-permissions) to these projects.
-  ![Invite teammates and set permissions form](./workspace-members-guests.png "Invite teammates and set permissions form")
+2. Go to the **Members** tab in the left panel and click on **Invite**
+   ![Invite people to Mergin Maps workspace](./dashboard-workspace-invite.jpg "Invite people to Mergin Maps workspace")
 
-4. Click on **Invite** to send an invitation. An email with a link will be sent to the recipients. After they accept the invitation, they will become members or guests of your workspace.
+3. In the form, enter the email addresses of people you want to invite and choose their [**workspace role**](./permissions/#workspace-member-roles) from the list.
+   - If you want to grant them access to all projects in the workspace, select one of the member roles (*Reader*, *Writer*, *Admin* or *Owner*).
+   - If you want to invite them only to some projects, select the *Guest* option and specify which projects should be shared with them.
```

Comment:
Edit text:
If you only want to invite them to some projects, select the *Guest* option and specify which projects should be shared with them.